tests/test_rpc.py

```diff
@@ -599,7 +599,9 @@ def all_spent(coins):
     with pytest.raises(RpcError, match="Insane timestamp.*"):
         lianad.rpc.startrescan(future_timestamp)
     assert lianad.rpc.getinfo()["rescan_progress"] is None
-    prebitcoin_timestamp = 1231006505 - 1
+    block_hash = bitcoind.rpc.getblockhash(0)
+    genesis_timestamp = bitcoind.rpc.getblock(block_hash)["time"]
+    prebitcoin_timestamp = genesis_timestamp - 1
     with pytest.raises(RpcError, match="Insane timestamp."):
         lianad.rpc.startrescan(prebitcoin_timestamp)
```

Comment:
This is not directly related to this PR, but perhaps we should also check that the rescan succeeds if starting from `genesis_timestamp`.

Reply:
adressed